script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -538,6 +540,7 @@ I should be safe now that I've put this much distance between us.
 -- Page 37, Offset 16664.
 -- 秋葉も登校すると言って飛び出した兄貴を追いかけてくるほど子供じゃないだろう。
 // I think that 登校する applies to Shiki here.
+// to be rechecked, something about the phrasing
```

Comment:
Question is about whether the sentence is parsed as 
> (秋葉も) (登校すると言って飛び出した兄貴) を追いかけてくるほど子供じゃないだろう

or 
> (秋葉も登校すると言って) (飛び出した兄貴) を追いかけてくるほど子供じゃないだろう

Reply:
Akiha said she would be going to school (now) too and she's probably not childish enough to give chase after her older brother if he legs it

Reply:
Yeah, Petrikow convinced me in vc of 
> (秋葉も登校すると言って) (飛び出した兄貴) を追いかけてくるほど子供じゃないだろう

In other words, it ought to be something like:
> Akiha isn't childish enough to announce 'I'm going to school too!' and then chase after her older brother as he legs it.